legal-api/report-templates/certificateOfAmalgamation.html

```diff
@@ -39,7 +39,7 @@
                     <div class="cert-separator"></div>
                     <div class="cert-issued-by">
                         <div>Issued under my hand at Victoria, British Columbia</div>
-                        <div>on {{ report_date_time }}</div>
+                        <div>on {{ effective_date }}</div>
```

Comment:
FYI. In incorporation filing and amalgamation filing the `effective_date` will be same as `business.formatted_founding_date`

Reply:
I was not asking you change. Just want you to know both are correct. Anyways looks good to me